bvm/ballerina-config/src/main/java/org/ballerinalang/config/ConfigProcessor.java

```diff
@@ -152,28 +185,42 @@ private static String getConfigFile(String fileLocation, Path defaultLocation) {
     }
 
     private static BConfig parseConfigFile(String path) throws IOException {
-        ANTLRFileStream configFileStream = new ANTLRFileStream(path);
+        InputStream inputstream = new FileInputStream(path);
+        Toml configToml = new Toml().read(inputstream);
```

Comment:
Need to catch the `IllegalStateException` and `RuntimeException` thrown here and wrap them in Ballerina errors.

Reply:
Added exception handlers for config file read failures.